libview/ev-view.c

```diff
@@ -3516,6 +3521,10 @@ ev_view_scroll_event (GtkWidget *widget, GdkEventScroll *event)
 		        case GDK_SCROLL_LEFT:
 				ev_view_previous_page (view);
 				break;
+#if GTK_CHECK_VERSION (3, 0, 0)
+                        case GDK_SCROLL_SMOOTH:
+                                g_assert_not_reached ();
```

Comment:
Well, i can look at upstream if there are some followers.
But, if i understand upstream commit correct is that a dummy function to fix the build warning.